packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Build a custom sign-in UI in your mobile app
+excerpt: Learn how to build a custom sign-in UI in your mobile app.
+layout: Guides
+---
+
+You can connect your mobile app to Okta and sign users in by [opening a browser](/docs/guides/sign-into-mobile-app/), however, if you prefer that your users not leave your app, you need to build a custom sign-in UI with native controls and screens instead. Use this guide to build a customized sign-in experience inside your mobile application.
+
+> **Note:** If the browser sign-in method works for your application, we recommend using that since building a custom sign-in UI takes more effort and development time.
+
+---
+
+**Learning outcomes**
+
+* Add the required packages to your application.
+* Build a custom mobile sign-in UI.
+* Handle the authentication responses appropriately.
+
+**What you need**
+
+* An Okta Developer organization. Don't have one? [Create one for free](https://developer.okta.com/signup)
+* A mobile app that you want to add a custom sign-in UI to
```

Comment:
```suggestion
* A mobile app with Okta authentication (see [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/)) that you want to add custom the sign-in UI to
```
Maybe merge these 2 points? The article references [Sign users in to your mobile app] and assumes that Okta integration app exists and that auth is already working.